README.md

```diff
@@ -0,0 +1,8 @@
+Add a separate category of the ramp, in order to achieve the effect of different shaderpacks through the separate configuration of the luminous ramp.
```

Comment:
We don't store changelogs in readme file. Put all this text in PR description, if you want to present the list of changes.